src/function.ts

```diff
@@ -1,10 +1,13 @@
 import fs from "fs";
 import path from "path";
 import * as appsync from "@aws-cdk/aws-appsync-alpha";
+
+// @ts-ignore - imported for typedoc
 import { serializeFunction } from "@functionless/nodejs-closure-serializer";
```

Comment:
use `import type` then to ensure it is not included in the physical module.

Reply:
I add them so that links in docs can still be used

Reply:
Was this meant to be on a different line? Looks like eslint may have moved it.

Reply:
Ah, eslint probably sorted and left it where it was.